server_installation/topics/installation/installing-rpm.adoc

```diff
@@ -21,7 +21,7 @@ Using Red Hat Subscription Manager, subscribe to the {appserver_name} {appserver
 
 [source,bash,subs="attributes+"]
 ----
-subscription-manager repos --enable=jb-eap-{appserver_version}-for-rhel-<RHEL_VERSION>-server-rpms
+subscription-manager repos --enable=jb-eap-7.0-for-rhel-<RHEL_VERSION>-server-rpms --enable=rhel-<RHEL_VERSION>-server-rpms
```

Comment:
{appserver_version} is a variable that will be replaced with the appropriate EAP release number when displayed in the docs. Since we refer to this EAP release version in many places, using the variable allows us to update the docs much more easily than hard-coding the number would allow. Please do a quick update here to put the variable back in.

Reply:
Yeah, you're right. Done ;)

Reply:
Thank you!